### PR TITLE
[#38] main menu

### PR DESCRIPTION
**Pull request by mfrydrychowicz**
Ok, So first of all I'm sorry that this is combo commit. Also, test case is quite simple, but I think we should focus now on delivering full functionality and bugfixes.
Nevertheless I am open for suggestions 😃

### CODE REVIEW

**Comment by mfrydrychowicz** [comment]
Also white button is to be connected (not finished yet). TODO: sections to be removed

File: package.json [hunk 1]
```diff
@@ -25,6 +25,7 @@
   },
   "homepage": "https://github.com/CodersCamp2020/CodersCamp2020.Project.JavaScript.StarWarsQuiz#readme",
   "dependencies": {
+    "modes": "^0.0.2",
```
Comment (mfrydrychowicz) [inline package.json:28]: to remove

File: styles/components/ranking.scss [hunk 2]
```diff
@@ -1,3 +1,27 @@
...
+  justify-content: center;
+  align-items: center;
+  font-weight: bold;
+  font-size: 2rem;
+}
+
+.icon-header > p {
+  margin-left: 2rem;
+}
+
```
Comment (mfrydrychowicz) [inline styles/components/ranking.scss:24]: was moved to box.scss and iconHeader. This is an issue caused by rebase. I will fix this

File: index.html [hunk 3]
```diff
@@ -13,12 +13,18 @@
 <body>
 
 <div id="swquiz-app" class="swquiz-app">
-    <div class="swquiz-header">
+    <div class="swquiz-row" id="swquiz-header">
+        <a href="/index.html">
+            <img class="swquiz-sw-image" src="./static/assets/ui/StarWarsLogo.png" alt="Star Wars Logo">
+        </a>
```
Comment (budnickip) [inline index.html:19]: maybe later it's best way to put logo with JS DOM?

File: src/app/App.js [hunk 4]
```diff
@@ -68,28 +62,21 @@ const testDataComputer = {
...
+  mainWindow();
+
+  // modalGameOver(
+  //   document.querySelector('#swquiz-app'),
+  //   testDataHuman,
+  //   testDataComputer,
+  //   modalSubmitCallback,
+  // );
+  // startTime(options.quizMaxTime)
+  // generateQuestion('people').then(res=> console.log(res))
```
Comment (budnickip) [inline src/app/App.js:81]: i always want to write this to you: "leftover - to remove"

File: src/app/gameMode.js [hunk 5]
```diff
@@ -1,5 +1,13 @@
...
+
+export const VEHICLE_MODE_QUESTION_STRING =
+  'MODE: Do you recognize this vehicle?';
+export const PEOPLE_MODE_QUESTION_STRING = 'MODE: Who is this Character?';
+export const STARSHIPS_MODE_QUESTION_STRING =
+  'MODE: Do you recognize this starship?';
+
 export const gameMode = (text) => {
-  const app = document.querySelector('#swquiz-app');
+  const app = document.getElementById('swquiz-game-header');
```
Comment (budnickip) [inline src/app/gameMode.js:10]: as I good remember querySelector is newer and better to use them to get elements from DOM.

**Review by budnickip — APPROVED** [review]

File: src/app/App.js (hunk 4 above)
Comment (mfrydrychowicz) [reply src/app/App.js:81]: hahaha I was thinking about removing this, but at the same time I wanted to save this code, for the person who will integrate all pieces 😄

**Pull request merged**